--- gts-0.7.3/src/rounding.h~	2004-10-11 00:51:53.000000000 +0200
+++ gts-0.7.3/src/rounding.h	2004-11-07 02:38:26.000000000 +0100
@@ -22,12 +22,17 @@
 #ifdef HAVE_FPU_CONTROL_H
 #  include <fpu_control.h>
 #  ifdef _FPU_EXTENDED
-     static fpu_control_t fpu_round_double =
-       (_FPU_DEFAULT & ~ _FPU_EXTENDED)|_FPU_DOUBLE;
-     static fpu_control_t fpu_init;
-#    define FPU_ROUND_DOUBLE  { _FPU_GETCW(fpu_init);\
-                                _FPU_SETCW(fpu_round_double); }
-#    define FPU_RESTORE       {_FPU_SETCW(fpu_init);}
+#    ifndef __alpha__
+       static fpu_control_t fpu_round_double =
+         (_FPU_DEFAULT & ~ _FPU_EXTENDED)|_FPU_DOUBLE;
+       static fpu_control_t fpu_init;
+#      define FPU_ROUND_DOUBLE  { _FPU_GETCW(fpu_init);\
+                                  _FPU_SETCW(fpu_round_double); }
+#      define FPU_RESTORE       {_FPU_SETCW(fpu_init);}
+#    else
+#      define FPU_ROUND_DOUBLE
+#      define FPU_RESTORE
+#    endif
 #  else /* not FPU_EXTENDED */
 #    define FPU_ROUND_DOUBLE
 #    define FPU_RESTORE
