--- gts-0.7.6/configure.in.orig	2006-03-10 00:48:36.000000000 +0100
+++ gts-0.7.6/configure.in	2013-01-20 17:03:21.689254791 +0100
@@ -43,7 +43,7 @@
 AM_INIT_AUTOMAKE($PACKAGE, $VERSION, no-define)
 
 # Specify a configuration file
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 
 AC_DEFINE_UNQUOTED(GTS_MAJOR_VERSION, $GTS_MAJOR_VERSION, [Major version])
 AC_DEFINE_UNQUOTED(GTS_MINOR_VERSION, $GTS_MINOR_VERSION, [Minor version])
